### DISCUSSION TITLE
Completely Uninstall Alltalk_tts

### DISCUSSION THREAD

**Discussion by WarmRainIdea** [erew123/alltalk_tts#173] [Q&A] [answered]
Hello, may I know how to completely uninstall alltalk_tts? Thank you.

**Comment by erew123** [comment, marked as answer]
Hi @WarmRainIdea 

Yes, its delete the `alltalk_tts` folder and all its contents. That will remove alltalk and any Python environment you have installed if you use a Standalone Installation.

Beyond that if you have have used Whisper of any Huggingface related things, you will need to delete your huggingface cache https://huggingface.co/docs/huggingface_hub/guides/manage-cache

Thanks

**Reply by WarmRainIdea** [reply]
Thank you very much!
